packages/safe-apps-provider/package.json

```diff
@@ -25,7 +25,7 @@
   },
   "homepage": "https://github.com/gnosis/safe-apps-sdk#readme",
   "dependencies": {
-    "@gnosis.pm/safe-apps-sdk": "7.5.0",
+    "@gnosis.pm/safe-apps-sdk": "file:.yalc/@gnosis.pm/safe-apps-sdk",
```

Comment:
senor could you remove yalc? yarn will use local packages so no yalc is needed

Reply:
yes yes, let me review it again because in the daily i commented it was ready but i had to do the last check as i changed some things this morning :D